src/main/java/launcher/Lehrry.java

```diff
@@ -1,7 +1,23 @@
 package launcher;
 
+import listeners.MainEventListener;
+import net.dv8tion.jda.api.JDA;
+import net.dv8tion.jda.api.JDABuilder;
+import net.dv8tion.jda.api.requests.GatewayIntent;
+import net.dv8tion.jda.api.utils.ChunkingFilter;
+
+import javax.security.auth.login.LoginException;
+
 public class Lehrry {
     public static void main(String[] args) {
-        System.out.println("hey!");
+        try {
+            JDA jda = JDABuilder.createDefault(System.getenv("LEHRRY_TOKEN"))
+                    .setChunkingFilter(ChunkingFilter.ALL)
+                    .enableIntents(GatewayIntent.GUILD_MESSAGES)
+                    .addEventListeners(new MainEventListener())
+                    .build();
+        } catch (LoginException ex) {
+            ex.printStackTrace();
```

Comment:
Yep, I'll create an issue about better error handling later. This reminds me that we also need to make an issue about input validation, since I don't think user input is actually validated for any of the commands.

Reply:
Now that I've looked at the code again, it appears that most input *is* validated, but no errors are displayed to the user. I'll still make an issue for it.

Reply:
Included in #10.